### PR TITLE
A+C: add Mark Skilbeck

### PR DESCRIPTION
**Pull request by notmgsk**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -102,3 +102,4 @@ Yevgeniy Vahlis <evahlis@gmail.com>
 Yucheng Zhu <zyctc000@gmail.com>
 Zoe Juozapaitis
 Patrick DeVivo <patrick@tickgit.com>
+Mark Skilbeck <markskilbeck@gmail.com>
```
Comment (kortschak) [inline AUTHORS:105]: I'll fix up the author above in another PR, but can you put your name in alphabetically, please.

Comment (notmgsk) [reply AUTHORS]: apologies

Comment (kortschak) [reply AUTHORS]: There were counter examples in the file that would lead you astray, approved by me, no apologies necessary.

File: AUTHORS [hunk 2]
```diff
@@ -101,4 +102,4 @@ Vladimír Chalupecký <vladimir.chalupecky@gmail.com>
 Yevgeniy Vahlis <evahlis@gmail.com>
 Yucheng Zhu <zyctc000@gmail.com>
 Zoe Juozapaitis
-Patrick DeVivo <patrick@tickgit.com>
+Patrick DeVivo <patrick@tickgit.com>
```
Comment (kortschak) [inline AUTHORS:105]: Sorry. Can you repair this and in CONTRIBUTORS. It will be changed later, but just to minimised diff here.

Comment (notmgsk) [reply AUTHORS]: That was careless of me.

**Review by kortschak — APPROVED** [review]
LTGM, thanks.

I'll merge this with #1535. I would like to wait a bit to see if anyone has any concerns about the naming (I doubt there will be).

**Pull request merged**